lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts

```diff
@@ -128,6 +128,10 @@ export class ContentNodeSelectorPanelComponent implements OnInit, OnDestroy {
     @Input()
     pageSize: number = this.DEFAULT_PAGINATION.maxItems;
 
+    /** Define the selection mode for document list. The allowed values are single or multiple */
+    @Input()
+    selectionMode: string = 'single';
```

Comment:
I suggest restricting the type so that typescript validates it better
```suggestion
    selectionMode: 'single' | 'multiple' = 'single';
```